spec/registry.md

```diff
@@ -833,3 +818,25 @@ For more information, see [Working with Timezones](https://w3c.github.io/timezon
 > The form of these serializations is known and is a de facto standard.
 > Support for these extensions is expected to be required in the post-tech preview.
 > See: https://datatracker.ietf.org/doc/draft-ietf-sedate-datetime-extended/
+
+### Date and Time Locale Options
+
+**_<dfn>Date/time locale options</dfn>_** override options set by the current locale
+or an implementation-defined date/time _operand_ value.
```

Comment:
I think this sentence is incomplete. I'm also still not fond of saying "locale" here (because I spend a lot of my life saying "time zone is orthogonal to locale"). Perhaps call these "date/time override options"? Even if not, the sentence should be correct.

```suggestion
**_<dfn>Date/time override options</dfn>_** are _options_ that allow an _expression_ to
override values set by the current locale,
or provided by the _formatting context_ (such as the default time zone),
or embedded in an implementation-defined date/time _operand_ value.
```